examples/README.md

```diff
@@ -0,0 +1,6 @@
+# Examples
+
+This directory contains example usage of smt-switch.
+
+## QF_UFBV
```

Comment:
can u explain here how one can change the cpp recompile and rerun?

Reply:
Great idea -- done with c2b862a13b2101020b66ded1942118f9002f718f.